Assets/Scenes/PerformanceStressTestSample/PerformanceStressTest.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
   m_CustomReflection: {fileID: 0}
   m_Sun: {fileID: 1526980366}
-  m_IndirectSpecularColor: {r: 1.1158099, g: 1.0351701, b: 1.0710709, a: 1}
+  m_IndirectSpecularColor: {r: NaN, g: Infinity, b: NaN, a: 1}
```

Comment:
Are those supposed to be Nan and Infinity? I didn't know that was a thing one could do.

Reply:
That's super weird. 

I've changed the values back. 

I pretty much never change the position of things (or their assigned colors for that matter) within the editor - I wonder whether this was Unity trying to be useful somehow. 🤷

Reply:
I'm assuming it was random Unity math errors.